dts/riscv/ite/it8801-mfd-gpiocr.dtsi

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2024 ITE Corporation. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	soc {
```

Comment:
This does not look quite right, how is this dtsi going to be used? the child nodes here should be child nodes of the mfd device, right?

Reply:
Yes, this is the child node that mfd will use. Renamed.